### DISCUSSION TITLE
"Other: Default action in the Alt+F4 dialog" seems to be ignored after Win11 update

### DISCUSSION THREAD

**Discussion by ScruffR** [valinet/ExplorerPatcher#3591] [Q&A]
I have several machines running Windows 11 Enterprise (KB5041585)
Although I have set `Default action in the Alt+F4 dialog` to `Restart` whenever I actually hit <kbd>Alt</kbd>+<kbd>F4</kbd> I get confronted with the `Switch user` option.

I also played with the registry setting 
```
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"Start_PowerButtonAction"=dword:00000004
```
but that also had no effect.

**Comment by pyrates999** [comment]
Where in the EP properties application did you set this?

**Comment by ScruffR** [comment]
As mentioned in the topic title: 
![image](https://github.com/user-attachments/assets/e5003424-e369-4a24-b2b7-9f3cdb7f8ec2)

**Comment by pyrates999** [comment]
Since this sets a registry entry, can you uninstall EP and just set that registry entry, stop explorer.exe and run explorer.exe, and see if it is working or not.

If it is still not working, MS may have disabled the code that enables this feature.

**Comment by ScruffR** [comment]
It seems MS had disabled it but after installing the latest update it started working again.

**Reply by pyrates999** [reply]
this one? https://support.microsoft.com/en-us/topic/october-22-2024-kb5044380-os-builds-22621-4391-and-22631-4391-preview-9a8db86a-815d-470a-be95-1aa2ad1feb05
